src/lib/stores/busy.store.ts

```diff
@@ -29,6 +40,10 @@ const initBusyStore = () => {
         state.filter(({ initiator }) => initiator !== initiatorToRemove)
       );
     },
+
+    reset() {
```

Comment:
In `nns-dapp` some stores have `resetForTesting()`. What do you think about having the same name, to make its role more obvious?

Reply:
Makes sense, thanks! Done.